Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [PolyMC/PolyMC](https://hosted.weblate.org/projects/polymc/polymc/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widgets/polymc/-/polymc/horizontal-auto.svg)

Commit:
Translated using Weblate (Portuguese (Brazil))

Currently translated at 100.0% (1030 of 1030 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/pt_BR/

Grooty12 committed:
Translated using Weblate (Danish)

Currently translated at 52.0% (536 of 1030 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/da/

Rafii2198 committed:
Translated using Weblate (Polish)

Currently translated at 99.8% (1028 of 1030 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/pl/

K0-RR committed:
Translated using Weblate (Polish)

Currently translated at 99.8% (1028 of 1030 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/pl/

K0-RR committed:
Translated using Weblate (Polish)

Currently translated at 100.0% (1030 of 1030 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/pl/

K0-RR commented on pl.po:
```diff
@@ -405,6 +406,10 @@ msgid ""
 "\n"
 "Please update both your operating system and PolyMC."
 msgstr ""
+"Konta Microsoft są jedynie dostępna na systemach macOS 10.13 lub nowszych, "
```
"jedynie dostępna"...

K0-RR commented on pl.po:
```diff
@@ -1923,6 +1928,10 @@ msgid ""
 "\n"
 "Do you want to play the demo?"
 msgstr ""
+"Te konto nie posiada zakupionego Minecrafta.\n"
```
And another one... "Te konto"

Scrumplex commented:
@RDKRACZ Is something wrong in the translations? If so, you can fix it on Weblate

K0-RR commented:
@Scrumplex those are spelling errors made by someone who didn't read even once what they have typed.
 
What does "&" mean and why it's inside a word? I have seen it a few times already and not all translations contain it.
![Screenshot from 2022-03-14 13-33-25](https://user-images.githubusercontent.com/43277609/158173716-69129e2e-8d44-4d07-bf92-3484de0f3380.png)

Scrumplex commented:
Not sure, it's also present in the source code: https://github.com/PolyMC/PolyMC/blob/develop/launcher/ui/widgets/CustomCommands.ui#L32

K0-RR committed:
Translated using Weblate (Italian)

Currently translated at 99.1% (1016 of 1025 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 99.1% (1016 of 1025 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (1025 of 1025 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/it/

Commit:
Translated using Weblate (Italian)

Currently translated at 100.0% (1025 of 1025 strings)

Translation: PolyMC/PolyMC
Translate-URL: https://hosted.weblate.org/projects/polymc/polymc/it/
